.github/workflows/ci.yml

```
@@ -363,19 +580,180 @@ jobs:
+        if: always()
+        run: |
+          echo "${{ steps.status-report.outputs.summary-report }}" >> $GITHUB_STEP_SUMMARY
+          
+      - name: Comment on PR with Status Report
+        if: always() && github.event_name == 'pull_request'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const statusReport = `${{ steps.status-report.outputs.summary-report }}`;
+            const overallStatus = '${{ steps.status-report.outputs.overall-status }}';
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)